src/govuk/components/pagination/template.njk

```diff
@@ -0,0 +1,76 @@
+{% set type = params.type | default("results") %}
+{% set blockLevel = not params.items and params.next and params.previous %}
+
+
+<nav class="govuk-pagination {{- ' ' + params.classes if params.classes }} {{- ' govuk-pagination--block' if blockLevel }}" role="navigation" aria-label="{{type}}">
+  {%- if params.previous and params.previous.href -%}
+    <div class="govuk-pagination__prev">
+      <a class="govuk-link govuk-pagination__link" href="{{ params.previous.href }}" rel="prev">
+        <span class="govuk-pagination__link-title {{- ' govuk-pagination__link-title--decorated' if blockLevel and not params.previous.label }}">
+          <svg class="govuk-pagination__icon govuk-pagination__icon--prev" xmlns="http://www.w3.org/2000/svg" height="13" width="15" aria-hidden="true" focusable="false" viewBox="0 0 15 13">
+            <path d="m6.5938-0.0078125-6.7266 6.7266 6.7441 6.4062 1.377-1.449-4.1856-3.9768h12.896v-2h-12.984l4.2931-4.293-1.414-1.414z"></path>
+          </svg>
+          {%- if params.previous.text -%}
+            {{- params.previous.text -}}
+          {%- else -%}
+            Previous<span class="govuk-visually-hidden">&nbsp;page</span>
+          {%- endif -%}
+        </span>
+
+        {%- if params.previous.label -%}
+          <span class="govuk-visually-hidden">:</span>
+          <span class="govuk-pagination__link-label">{{ params.previous.label }}</span>
+        {%- endif -%}
+      </a>
+    </div>
+  {% endif %}
+
+  <ul class="govuk-pagination__list">
```

Comment:
At the minute this is outputted even if there are no items, so e.g. in the `with-prev-and-next-only` example there's an empty `<ul></ul>`

Reply:
I forgot to put an `if` there! Will sort out.